ruby/test/integration/minitest_redis_test.rb

```diff
@@ -115,18 +116,18 @@ def test_junit_reporter
       end
       assert_empty err
       output = normalize(out.lines.last.strip)
-      assert_equal 'Ran 8 tests, 5 assertions, 1 failures, 1 errors, 1 skips, 3 requeues in X.XXs', output
+      assert_equal 'Ran 7 tests, 4 assertions, 1 failures, 1 errors, 1 skips, 2 requeues in X.XXs', output
```

Comment:
Unless I'm mistaken, the changes here is because the flaky test isn't requeued anymore.